addons/website/views/website_templates.xml

```diff
@@ -1662,7 +1662,14 @@
 
             This also allows to gain some page speed scoring.
             -->
-            <span t-field="website.logo" t-options="{'widget': 'image', 'width': 95, 'height': 40}" role="img" t-att-aria-label="'Logo of %s' % website.name" t-att-title="website.name"/>
+            <t t-if="editable">
+                <!-- In editor: avoid fixed width/height so live height changes preview correctly. -->
+                <span t-field="website.logo" t-options="{'widget': 'image'}" role="img" t-att-aria-label="'Logo of %s' % website.name" t-att-title="website.name"/>
+            </t>
+            <t t-else="">
+                <!-- In public: reserve space to reduce CLS with width/height attributes. -->
+                <span t-field="website.logo" t-options="{'widget': 'image', 'width': 95, 'height': 40}" role="img" t-att-aria-label="'Logo of %s' % website.name" t-att-title="website.name"/>
+            </t>
```

Comment:
Why not use t-att-arial-label in editable ?

Maybe use t-set to compute the value for t-options ?
<t t-set="options" t-esc=" if editable:  '{'widget': 'image'}' : else '{'widget': 'image', 'width': 95, 'height': 40}' ">